fusedrug_examples/interaction/drug_target/affinity_prediction/bimodal_mca_PPI/scripts/run_simple_hp_grid_search.py

```diff
@@ -0,0 +1,203 @@
+import sys
```

Comment:
both this and the session manager one are too tied with LSF, we should consider removing them